lib/event_sourcery/aggregate_root.rb

```diff
@@ -10,15 +60,32 @@ def self.included(base)
     end
 
     module ClassMethods
+      # Collection of event handlers for the events that this aggregate cares about
+      #
+      # @return Hash
       attr_reader :event_handlers
 
+      # Register an event handler for the specified event(s)
+      #
+      # @param event_classes one or more event types for which the handler is for
+      # @param block the event handler
+      #
+      # @example
+      #  apply TodoAdded do |event|
+      #    @added = true
+      #  end
       def apply(*event_classes, &block)
         event_classes.each do |event_class|
           @event_handlers[event_class.type] << block
         end
       end
     end
 
+    # Load an aggregate instance based on the given ID and events
+    #
+    # @param id [Integer] ID of the aggregate instance to be loaded
```

Comment:
I thought aggregate ID should be a uuid respresented as a string. Surprised to see that it is an integer.